CONTRIBUTING.md

```diff
@@ -28,15 +28,14 @@ The BadgingAPI is built by the community and warmly welcomes collaboration. So a
 Before cloning this repository, make sure you have the latest versions;
 
 - [NodeJS and NPM](https://nodejs.org/en/download)
-- [MySQL](https://dev.mysql.com/downloads/installer/)
+- [MySQL](https://www.digitalocean.com/community/tutorials/how to-install-mysql-on-ubuntu-22-04)
```

Comment:
@peshatuhura it is better to keep the https://dev.mysql.com/downloads/installer/ because it is the main link of `MySQL` meaning all other links to MySQL depend on it. Consider it as the main source of information and updates where the rest of the links have first come to check out whether something changed before they write their articles.